.github/workflows/tagPublish.yml

```diff
@@ -0,0 +1,86 @@
+name: package and publish to pypi
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  github-release:
+    name: Create Github release page
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.release.tag_name }}
+      - name: Set release version
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: Create Github release page
+        # Create the release page first or the nodejs artifacts upload will fail
+        run: gh release create ${{ env.RELEASE_VERSION }} --draft --notes "Auto created by tagPublish workflow."
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  wheel:
+    name: python${{ matrix.python-version }}-${{ matrix.platform.python-architecture }} ${{ matrix.platform.os }}
+    runs-on: ${{ matrix.platform.os }}
+    strategy:
+      matrix:
+        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
+        platform: [
+        { os: "ubuntu-latest",  python-architecture: "x64", rust-target: "x86_64-unknown-linux-gnu" },
+        { os: "macOS-latest",   python-architecture: "x64", rust-target: "x86_64-apple-darwin" },
+        { os: "windows-latest", python-architecture: "x64", rust-target: "x86_64-pc-windows-msvc" },
+        { os: "ubuntu-20.04",  python-architecture: "x64", rust-target: "x86_64-unknown-linux-gnu" },
+        # { os: "windows-latest", python-architecture: "x86", rust-target: "i686-pc-windows-msvc" },
```

Comment:
Remove this commented code if not required.

Reply:
Done.